install/src/main/kotlin/org/jetbrains/bsp/bazel/install/InstallationContextProvider.kt

```diff
@@ -28,7 +28,9 @@ object InstallationContextProvider {
     fun generateAndSaveProjectViewFileIfNeeded(cliOptions: CliOptions) {
         val generatedProjectViewFilePath = calculateGeneratedProjectViewPath(cliOptions)
         if (!generatedProjectViewFilePath.isFileExisted() || cliOptions.projectViewCliOptions != null) {
-            ProjectViewCLiOptionsProvider.generateProjectViewAndSave(cliOptions, generatedProjectViewFilePath)
+            if (!generatedProjectViewFilePath.exists()) {
```

Comment:
I think I missed this one: Why do we need to extra guard this?

Reply:
That was mostly for the tests, but also if `generatedProjectViewFilePath.isFileExisted() ` is true and `cliOptions.projectViewCliOptions != null` is true, this will throw an exception. That popped up in the added e2e test, but also it seems sensible not to try and create a file if it already exists.